Sources/ProjectSpec/Target.swift

```diff
@@ -32,6 +32,7 @@ public struct Target: ProjectTarget {
     public var directlyEmbedCarthageDependencies: Bool?
     public var requiresObjCLinking: Bool?
     public var prebuildScripts: [BuildScript]
+    public var postCompileScripts: [BuildScript]
```

Comment:
Now the old `prebuildScripts` and `postbuildScript` names look funny. Maybe we should update them to `preBuildScripts` and `postBuildScript`, and use the old casing as a fallback?
What do you think?

Reply:
Sounds good.